src/components/app/data/queries/queryKeyFactory.js

```diff
@@ -86,6 +86,10 @@ const enterprise = createQueryKeys('enterprise', {
             queryKey: [availableCourseRunKeys],
             queryFn: async ({ queryKey }) => fetchCanRedeem(queryKey[2], availableCourseRunKeys),
           }),
+          canUpgradeWithLearnerCredit: {
```

Comment:
[curious] Could we make use of the existing `canRedeem` query? See related comment above.